lib/l10n/intl_es.arb

```
@@ -894,6 +899,60 @@
+  "@failedSendMessage": {
+    "placeholders": {
+      "error": { "type": "String" }
+    }
+  },
+  "waitingAdminAssignmentInput": "Esperando asignación de administrador...",
+  "typeYourMessage": "Escribe tu mensaje...",
+  
+  "@_comment_dispute_list": "Texto de Lista de Disputas",
+  "disputesWillAppear": "Las disputas aparecerán aquí cuando las abras desde tus intercambios",
+  "failedLoadDisputes": "Error al cargar disputas",
+  "retry": "Reintentar",
```

RuboCop (1.76.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)